lib/rubocop/cop/internal_affairs/undefined_config.rb

```diff
@@ -5,12 +5,21 @@ module Cop
     module InternalAffairs
       # Looks for references to a cop configuration key that isn't defined in config/default.yml.
       class UndefinedConfig < Base
+        # FileFinder is private API not expected to be used by cops but its fine for InternalAffairs
```

Comment:
```suggestion
        # `FileFinder` is a private API not intended to be used by cops, but it's fine for `InternalAffairs`.
```

Reply:
Put it on two lines because of LineLength